creator-node/src/services/stateMachineManager/stateReconciliation/recoverOrphanedData.jobProcessor.ts

```diff
@@ -194,7 +194,7 @@ const _saveWalletsWithThisNodeInReplicaToRedis = async (
   const numWalletsWithNodeInReplicaSet = await redisClient.scard(
     WALLETS_WITH_NODE_IN_REPLICA_SET_KEY
   )
-  logger.info(
+  logger.debug(
```

Comment:
can leave this as info i think, unless it is insanely frequent (it should be very rare?)